apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/transaction/messages/AbstractAppendix.java

```diff
@@ -27,10 +29,15 @@
         this.version = ((Number) attachmentData.get("version." + getAppendixName())).byteValue();
     }
 
+    @Deprecated(since = "TransactionV3")
```

Comment:
It will be deprecated, only when v3 version will be introduced. Not it is the only one correct and properly tested implementation

Reply:
removed all V3 implementations.